gradle.properties

```diff
@@ -18,10 +18,11 @@
 #####################################################################
 group=org.rundeck
 currentVersion = 3.4.10
-grailsVersion=4.0.3
-gormVersion=7.0.4.RELEASE
+grailsVersion=5.0.3
```

Comment:
The latest Grails release is 5.1.2. We probably want to upgrade to that to get the latest versions of dependencies.

We have a number of dependency version overrides set in this file. We should see how many of them we can remove. Ideally the Grails and Spring BOMs will specify the best versions of these dependencies and we should only override them for security purposes.

Reply:
Agree we should do it, i'd like to get the 5.0 work merged then do that